app/src/main/java/zegreanu/cristi/geoquiz/CheatActivity.kt

```diff
@@ -35,6 +36,9 @@ class CheatActivity : AppCompatActivity() {
             setAnswerShownResult(true)
             showAnimation()
         }
+
+        apiLevelTextView = findViewById(R.id.api_level_text_view)
+        apiLevelTextView.text = String.format("API Level ${Build.VERSION.SDK_INT}")
```

Comment:
Hmm, not sure String.format is needed :-?. See String interpolation from https://kotlinlang.org/docs/reference/idioms.html